src/veh_interact.cpp

```diff
@@ -572,13 +589,13 @@ void veh_interact::do_install()
     }
     mvwprintz(w_mode, 0, 1, c_ltgray, _("Choose new part to install here:"));
     wrefresh (w_mode);
-    
+
```

Comment:
Whitespace problems happen, I didn't catch it either.  Leastwise this is a (quasi) substantive merge conflict rather than two folks sticking JSON in the same place.